requirements.txt

```diff
@@ -0,0 +1 @@
+GitPython
```

Comment:
Please, fix current library version in requirements and add blank new line in the end

Reply:
Yes, thanks